tests/test_rtdc_check_dataset.py

```diff
@@ -291,6 +292,44 @@ def test_ic_metadata_empty_string():
     assert cues[0].cfg_key == "medium"
 
 
+def test_ic_fl_max():
```

Comment:
Could you please add one more test where there are no negative values, but values with "0.1"?